src/library/Uno.Material/Styles/Controls/ToggleSwitch.xaml

```diff
@@ -302,42 +358,66 @@
 
 						<Rectangle x:Name="OuterBorder"
 								   Grid.Row="2"
-								   Margin="2,0"
-								   Height="12"
-								   Width="40"
-								   RadiusX="6"
-								   RadiusY="6"
+								   Margin="10,0"
```

Comment:
I'm wondering if those values should be placed soon in a resource dictionary to enable a "compact" mode and touch modes, so that both modes can be used easily. (in a future PR)

Reply:
@jeromelaban thanks for the comment, I just want to be sure to understand the "compact"/"touch" modes. Do you have any documentation or specs I can refer to ?